implementations/rust/ockam/ockam/src/remote.rs

```diff
@@ -219,12 +219,10 @@ impl Worker for RemoteForwarder {
                 let route = msg.return_route();
 
                 info!("RemoteForwarder registered with route: {}", route);
-                let address;
-                if let Some(a) = route.clone().recipient().to_string().strip_prefix("0#") {
-                    address = a.to_string();
-                } else {
-                    return Err(OckamError::InvalidHubResponse.into());
-                }
+                let address = match route.clone().recipient().to_string().strip_prefix("0#") {
```

Comment:
Recently I found out the most convenient (for me) way to write such things:
```
let address = route
                    .clone()
                    .recipient()
                    .to_string()
                    .strip_prefix("0#")
                    .ok_or(OckamError::InvalidHubResponse)?
                    .to_string();
```
In case anyone likes this as well